### PR TITLE
Audiocodes Session Reassembly

### PR DESCRIPTION
**Pull request by Dletta**
Provide Re-Assembly when syslog for SIP is split across multiple syslog messages.

### CODE REVIEW

File: plugins/filters/app_audiocodes/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pastash/filter_app_audiocodes",
-  "version": "1.1.5",
+  "version": "1.1.6",
```
Comment (lmangani) [inline plugins/filters/app_audiocodes/package.json]: I would propose a 1.2.0 or even 2.0.0 due to the high number of changes 👍

File: plugins/filters/app_audiocodes/readme.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ Parameters for `app_audiocodes`:
 * `correlation_contact`: Auto-Extract correlation from Contact x-c. Default : false.
 * `debug`: Enable debug logs. Default : false.
 * `file_debug`: Enable debug using file input. (For development) Default : false.
-* `version`: Syslog parser version. Supports `7.40A.500` _(or higher)_. Default: 7.20A.260.012
+* `version`: Syslog parser version. Supports `7.40A.500` _(or higher)_. Default: 7.20A.260.012 (OPTIONAL after version 1.1.6)
```
Comment (lmangani) [inline plugins/filters/app_audiocodes/readme.md]: We can just completely remove this requirement and default to latest?

**Comment by Dletta** [comment]
Suggestions incorporated, ready for merge

**Comment by lmangani** [comment]
published  `+ @pastash/filter_app_audiocodes@1.2.0`

**Pull request merged**